packages/plugin-javascript/tests/utilities.test.ts

```diff
@@ -0,0 +1,351 @@
+import {Package} from '@sewing-kit/plugins';
+
+import {withWorkspace} from '../../../tests/utilities';
+import {
+  ExportStyle,
+  getLatestModifiedTime,
+  generateBabelPackageCacheValue,
+} from '../src/utilities';
+import {getModifiedTime, writeToSrc} from './utilities';
+
+describe('utilities', () => {
+  describe('getLatestModifiedTime()', () => {
+    it('gets the latest modified time', async () => {
+      await withWorkspace('simple-package', async (workspace) => {
+        const testPackage = new Package({
+          name: 'simple-package',
+          root: workspace.root,
+        });
+        const fileExtensions = ['.js'];
+
+        await writeToSrc(workspace, 'index.js');
+
+        const latestModifiedTime = getLatestModifiedTime(
+          testPackage,
+          fileExtensions,
+        );
+
+        await writeToSrc(workspace, 'index.js');
+
+        const updatedLatestModifiedTime = getLatestModifiedTime(
+          testPackage,
+          fileExtensions,
+        );
+
+        expect(updatedLatestModifiedTime).toBeGreaterThan(latestModifiedTime);
+      });
+    });
+
+    it('gets the latest modified time of a group of files', async () => {
+      await withWorkspace('simple-package', async (workspace) => {
+        const testPackage = new Package({
+          name: 'simple-package',
+          root: workspace.root,
+        });
+        const fileExtensions = ['.js', '.ts', '.mjs'];
+
+        await writeToSrc(workspace, 'index.js');
+        const indexModifiedTime = getModifiedTime(
+          testPackage.fs.resolvePath('src', 'index.js'),
+        );
+
+        await writeToSrc(workspace, 'file-a.ts');
+        const fileAModifiedTime = getModifiedTime(
+          testPackage.fs.resolvePath('src', 'file-a.ts'),
+        );
+
+        await writeToSrc(workspace, 'file-b.mjs');
+        const fileBModifiedTime = getModifiedTime(
+          testPackage.fs.resolvePath('src', 'file-b.mjs'),
+        );
+
+        const latestModifiedTime = getLatestModifiedTime(
+          testPackage,
+          fileExtensions,
+        );
+
+        expect(latestModifiedTime).toBeGreaterThan(indexModifiedTime);
+        expect(latestModifiedTime).toBeGreaterThan(fileAModifiedTime);
+        expect(latestModifiedTime).toEqual(fileBModifiedTime);
+      });
+    });
+
+    it('excludes files with extensions not included in the provided list', async () => {
```

Comment:
I could be missing something here. But how does this test description line up with the assertion?

Reply:
Oh, is it because this test is only building `.js` files and it's adding a `.ts` file to the workspace which shouldn't impact the build.

Reply:
Let's re-phrase the test description a bit then to make it easier for readers. 

Maybe: 
```
excludes .ts file extensions when it is not accounted for in fileExtensions 
```